bin/query-blocks.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Queries for the blocks on the site.
+ *
+ * @package Genesis\CustomBlocks
+ */
```

Comment:
This file is almost definitely not where this code will live. It's in a WP-CLI file just for the proof of concept.